libs/ledger-live-common/package.json

```diff
@@ -120,6 +120,8 @@
     "https": false
   },
   "dependencies": {
+    "@apollo/client": "^3.8.7",
```

Comment:
Do we need to keep the previous deprecated library ? (not sure if we removed it)